tiledb/ml/models/tensorflow_keras.py

```diff
@@ -385,3 +401,19 @@ def _load_weights_from_tiledb(
                 )
             var_value_tuples.extend(zip(weight_vars, read_weight_values))
         backend.batch_set_value(var_value_tuples)
+
+    def _get_tensorboard_files(self, log_dir: str) -> Mapping[str, bytes]:
+        """
+        This function accepts a Path log directory, evaluates path existense and returns a dictionary of TB log event files
+        :param log_dir: The directory that Tensorboard callback uses to store its files
+        :return DefaultDict: The DefaultDict[str, bytes] where key is the event file name and values are bytes of file data
+        """
+        event_files: DefaultDict[str, bytes] = defaultdict(bytes)
+        if os.path.exists(log_dir):
+            for file in os.listdir(f"{log_dir}/train"):
+                if "tfevents" not in file:
+                    continue
+                with open(f"{log_dir}/train/{file}", "rb") as f:
+                    data = f.read()
+                    event_files[f"{log_dir}/train/{file}"] = data
+        return event_files
```

Comment:
This method can be simplified with [glob.glob](https://docs.python.org/3/library/glob.html#glob.glob):
```py
        event_files = {}
        for path in glob.glob(f"{log_dir}/train/*tfevents*"):
            with open(path, "rb") as f:
                event_files[path] = f.read()
        return event_files
```